tests/functional/test_confirm_email_address.py

```diff
@@ -1,37 +1,31 @@
-import pytest
 from flask import url_for
-
-email_depts = [('some.one@digital.cabinet-office.gov.uk',
-                'Government Digital Service')]
+from tests.functional.user_flows import on_email_confirm_fill_in_then_continue
+from tests.functional.user_flows import yes_selected, no_selected
```

Comment:
Are these imports expected to be used across a number of test modules? I don't see any need for them to be in a separate file from the tests that use them if not.